index.d.ts

```diff
@@ -411,7 +411,7 @@ export interface SeparatorProps {
   tintColor?: string;
 }
 
-export class TableView extends React.Component<null> {}
+export class TableView extends React.Component {}
```

Comment:
When it's omitted, the type parameter is `{}` (empty object).
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/710943739f2313edc16526989c5ea83f0929b324/types/react/index.d.ts#L75